src/search/landmarks/landmark_graph.h

```diff
@@ -112,7 +113,8 @@ class LandmarkGraph {
        when moving landmark graph creation there. */
     int get_num_edges() const;
 
```

Comment:
We could also do something more sophisticated like counting how many reasonable / obedient reasonable orders we have if we are interested in this output.